package.json

```diff
@@ -6,23 +6,37 @@
   "scripts": {
     "watch": "webpack --watch --config webpack.dev.js",
     "build": "webpack --config webpack.prod.js",
-    "watch-poll": "webpack --watch --watch-poll --config webpack.dev.js"
+    "watch-poll": "webpack --watch --watch-poll --config webpack.dev.js",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix"
   },
   "author": "Event Espresso",
   "license": "GPL-2.0",
   "browserslist": [
     "extends @wordpress/browserslist-config"
   ],
+  "engines": {
+    "node": ">=8.0.0",
+    "npm": ">=5.0.0"
+  },
```

Comment:
missed mentioning this in commit notes, I _think_ this will ensure the right version of node/npm is being used.  I've already updated our `ee.devbox.bots` server so we should be okay there.